well-known-bots.json

```
@@ -8467,5 +8467,29 @@
       "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:72.0) Gecko/20100101 Firefox/72.0 DatadogSynthetics"
     ],
     "url": "https://docs.datadoghq.com/synthetics/guide/identify_synthetics_bots/?tab=singleandmultistepapitests#user-agent"
-  }
+  },
+  {
+    "id": "Hydrozen.io-user-agent",
+    "categories": [
+      "monitor"
+    ],
+    "pattern": "^Hydrozen\\.io/\\d+\\.\\d+$",
```

```suggestion
    "pattern": "^Hydrozen\\.io/\\d{0,4}\\.\\d{0,4}$",
```
What do we think about limiting to 4 digits in the version? Alternatively, we could just check the first portion